proposals/PlayServicesVersion_Proposal.md

```diff
@@ -0,0 +1,60 @@
+# Play Services Version Proposal
+
+- Status: Proposed
+
+## Overview
+This is a proposal to offer an alternative solution for [CB-13145](https://issues.apache.org/jira/browse/CB-13145).
+
+## Problem
+Plugins using different versions of Google Play Services or the `+` to indicate range cause gradle to error out with the following error:
+
+```
+> Failed to apply plugin [id 'com.google.gms.google-services']
+   > For input string: "+"
+```
+
+## Potential Solution
+Currently, the proposed solution is to create an Android only preference in `config.xml` to control the version of Google Play Services for the whole project. For instance, `config.xml` would look like this:
+`<preference name = "PlayServicesVersion" value = "11.0.0"/>`
+
+Then if, for example, the `phonegap-plugin-push` wanted to include FCM libraries in `plugin.xml`, it would contain:
+`<framework src="com.google.firebase:firebase-messaging:$PlayServicesVersion"/>`
+
+If the `cordova-plugin-google-analytics` wanted to include Play Services in its `plugin.xml` it would contain:
+`<framework src="com.google.android.gms:play-services-analytics:$PlayServicesVersion"/>`
+
+These plugins would both use the same version of Google Play Services. This gets around the problem of plugins using + in the framework tag and having two plugins pin different versions of Play Services.
```

Comment:
This gets around the problem of plugins using + in the framework tag (which isn't supported anymore) or having the two plugins pin different versions of Play Services (gradle errors with that)